### ISSUE TITLE
Error while running deploy

### ISSUE THREAD

**Issue by PingusPepan** [SnowdogApps/magento2-frontools#79]
Hey,
I found a bug in task/deploy.js
When deploying custom theme which is inheriting from sass-blank the deployment fails. The culprit is in lines 41 and 53 of the deploy.js file.
My server file structure contains the word web multiple times so I am getting this error:
```
Error: ENOENT: no such file or directory, symlink
/.../client1/web5/web/magento21/vendor/snowdog/theme-blank-sass/Magento_Swatches/web/images/loader-2.gif ->
/.../client15/web/magento21/vendor/snowdog/theme-blank-sass/Magento_Swatches/web/images/loader-2.gif
```
I shortened the file paths.
I am thinking of proper solution now. For now only thing that I came up with is to replace only last occurence of web in the path but there has to be some better solution I believe.

**Comment by PingusPepan** [comment]
I managed to work around it by changing line 41 to:
`const destPath = dest+srcPath.replace(parentSrc, '').replace('/web', '');`
and line 53 to:
`const destPath = dest+srcPath.replace(src, '').replace('/web', '');`

It's just a workaround but it worked for me!

**Comment by Igloczek** [comment]
Replacing only last occur, probably will be better option. I'll try to fix this in next version.

**Comment by Igloczek** [comment]
Will be released with 1.0.0
